src/templates/static/metatrader/download.jsx

```diff
@@ -1,108 +1,228 @@
+            <DownloadButton
+                badge={badge}
+                href={href}
+                download={download}
+                os={os}
+                target={target}
+            >
+                {it.L(`Download MT5 for ${title}`)}
+            </DownloadButton>
         </div>
+    </div>
+);
 
-        <SeparatorLine no_wrapper />
+const Download = () => {
+    const windows_link = `<a data-type='alt-link' data-target='windows' id='enable-windows-alt-link'>${it.L('Windows')}</a>`;
+    const linux_link = `<a data-type='alt-link' data-target='linux' id='enable-linux-alt-link'>${it.L('Linux')}</a>`;;
+    const mac_link = `<a data-type='alt-link' data-target='mac' id='enable-mac-alt-link'>${it.L('Mac')}</a>`;;
+    return (
+        <div id='mt_download' className='static_full center-text'>
+            <h1>{it.L('Start Trading with MetaTrader 5')}</h1>
+            <h3>{it.L('Trade with a powerful interface known as the global industry standard.')}</h3>
 
-        <div className='gr-padding-20'>
-            <h3>{it.L('For Mobile') }</h3>
-            <p>{it.L('Access the markets anytime, anywhere using native apps for your iOS or Android devices.') }</p>
-            <div className='gr-row'>
-                <div className='gr-2 gr-hide-m' />
-                <DownloadBlock
-                    image='android'
-                    os='android'
-                    title={it.L('Android')}
-                    desc={it.L('All versions of Android')}
-                    target='_blank'
-                    badge='google-play-badge'
-                />
-                <DownloadBlock
-                    image='apple'
-                    os='ios'
-                    title={it.L('iOS')}
-                    desc={it.L('All versions of iOS')}
-                    target='_blank'
-                    badge='app-store-badge'
-                />
+            <div className='gr-padding-20'>
+                <h1 className='invisible download-heading' id='windows-heading'>
+                    {it.L('MT5 for Windows')}
+                </h1>
+                <h1 className='invisible download-heading' id='macos-heading'>
+                    {it.L('MT5 for Mac')}
+                </h1>
+                <h1 className='invisible download-heading' id='linux-heading'>
+                    {it.L('MT5 for Linux')}
+                </h1>
+                <p>
+                    {it.L(
+                        'Download MT5 for your desktop or laptop to access the powerful tools and features enjoyed by millions of traders.'
+                    )}
+                </p>
+                <div className='gr-row'>
+                    <DesktopDownloadBlock
+                        id='windows'
+                        title={it.L('Windows')}
+                        image='windows.png'
```

Comment:
if you make the id and image have the  same name you don't need to pass it twice. the .png can be assumed as it's true for all of them right now.

Reply:
I suggest also using the title to derive the id